docs/Makefile.am

```
@@ -50,6 +50,7 @@ EXTRA_DIST =                                    \
  CIPHERS.md                                     \
  CMakeLists.txt                                 \
  CODE_OF_CONDUCT.md                             \
+ CODE_REVIEW.md					\
```

Spaces instead of tabs to match the others
```suggestion
 CODE_REVIEW.md                                 \
```